pywwt/ss_proxy.py

```diff
@@ -0,0 +1,81 @@
+from astropy import units as u
+from traitlets import HasTraits, observe, validate, TraitError
+
+from .traits import Bool, Int
+
+class SolarSystem(HasTraits):
+
+    def __init__(self, BaseWWTWidget):
```

Comment:
I would avoid using ``BaseWWTWidget`` here as that is the class name (not the instance) - instead use ``base_wwt_widget``